### DISCUSSION TITLE
Use docker image from GitHub Packages as container

### DISCUSSION THREAD

**Discussion by lambda-xyz** [community/community#25330] [Actions]
<p>I’d like to use a public Docker image available on GitHub Package in the field <code>jobs.&lt;job_id&gt;.container</code>:</p>
<pre><code class="lang-auto">name: CI

on:
  push:
    branches:
      - master

jobs:
  job_build:
    name: Build
    runs-on: ubuntu-latest

    container:
      image: docker.pkg.github.com/OWNER/REPOSITORY/IMAGE_NAME:latest

    steps:
      - name: Clone
        uses: actions/checkout@v2

      - name: Build
        run: |
          make

      - name: Upload bin
        uses: actions/upload-artifact@v2
        with:
          name: bin
          path: bin
</code></pre>
<p>I guess I need to put <code>${{ secrets.GITHUB_TOKEN }}</code> somewhere but there is no field for that in the doc: <a href="http://jobs.%3Cjob_id%3E.container" rel="nofollow noopener">https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainer</a>.</p>
<p>Is it even possible to do that?</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p><a class="mention" href="/lambda-xyz">@lambda-xyz</a><br>
I am afraid I have to tell you that GitHub Actions doesn’t support using image from GitHub Package Registry for <code>Container</code> currently.<br>
There are some other communities who asked for this feature before. And GitHub Engineering team is trying to find a way to provide credential to daemon when pulling image from registry.<br>
You could also ask for this feature in the <a href="https://support.github.com/contact/feedback?contact%5Bcategory%5D=actions" rel="nofollow noopener">Feedback form for GitHub Actions</a> to improve the priority of it.</p>

**Comment by lambda-xyz** [comment]
<p>Wow, this is a very basic feature…</p>
<p>Hope to see it soon!</p>
<p>Thanks.</p>
